beluga-tunnel/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+use beluga_mqtt::MqttClient;
+use error::Error;
+
+mod error;
+mod proto;
+mod services;
+mod tunnel;
+
+pub type Result<T> = core::result::Result<T, Error>;
+
+pub use tunnel::Tunnel;
+
+#[derive(Debug)]
+pub struct TunnelManager {
+    mqtt: MqttClient,
+}
```

Comment:
Can we remove the `beluga-mqtt` dependency from the tunnel lib entirely?
The reason why I wanted this to be a separate crate is because a secure tunnel can be used even without mqtt.
Basically I think `beluga-tunnel` should just be the websocket stuff and `beluga` should handle receiving the tunnel details over mqtt and starting the tunnel.

Reply:
that makes sense, let's do it.